src/main/java/net/mcreator/ui/init/ArmorMakerTexturesCache.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2020 Pylo and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.init;
+
+import net.mcreator.io.ResourcePointer;
+import net.mcreator.io.TemplatesLoader;
+import net.mcreator.io.UserFolderManager;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jetbrains.annotations.Nullable;
+
+import javax.imageio.ImageIO;
+import javax.swing.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class ArmorMakerTexturesCache {
+	private static final Logger LOG = LogManager.getLogger("Armor Texture Cache");
+	private static final String MCR_FOLDER = System.getProperty("user.home")+ "\\.mcreator\\";
+
+	public static final Map<String, ImageIcon> CACHE = new ConcurrentHashMap<>();
+	public static final List<String> NAMES = new ArrayList<>();

```

Comment:
No need for this, you can get key or value set from map directly. Also, make getters for these.